scripts/perftests/PerformanceTestUtilities.ps1

```diff
@@ -0,0 +1,153 @@
+    function Log([string]$logStatement, [string]$color)
+    {
+        if(-not ([string]::IsNullOrEmpty($color)))
+        {
+            Write-Host "$($(Get-Date).ToString()): $logStatement" -ForegroundColor $color
+        }
+        else
+        { 
+            Write-Host "$($(Get-Date).ToString()): $logStatement"
+        }
+    }
+
+    function GetAbsolutePath([string]$Path)
+    {
+        $Path = [System.IO.Path]::Combine(((pwd).Path), ($Path));
+        $Path = [System.IO.Path]::GetFullPath($Path);
+        return $Path;
+    }
+
+    function OutFileWithCreateFolders([string]$path, [string]$content){
+        $folder = [System.IO.Path]::GetDirectoryName($path)
+        If(!(test-path $folder))
+        {
+            & New-Item -ItemType Directory -Force -Path $folder > $null
+        }
+        Add-Content -Path $path -Value $content
+    }
+
+    function GetAllPackagesInGlobalPackagesFolder([string]$packagesFolder)
+    {
+        if(Test-Path $packagesFolder)
+        {
+            $packages = Get-ChildItem $packagesFolder\*.nupkg -Recurse
+            return $packages
+        }
+        return $null
+    }
+
+    function GetFiles([string]$folder)
+    {
+        if(Test-Path $folder)
+        {
+            $files = Get-ChildItem $folder -recurse
+            return $files
+        }
+        return $null
+    }
+
+    function GetNuGetFoldersPath()
+    {
+        $nugetFolder = [System.IO.Path]::Combine($env:UserProfile, "np")
```

Comment:
why `np`? I thought we used `.nuget`

Reply:
I'm redirecting all the nuget folders and caches to not stomp onto the local user's global packages folder. 

So np is just a random (stands for NuGetPerformance) placeholder.